addons/openebs/3.10.0/install.sh

```diff
@@ -190,6 +190,9 @@ function openebs_apply_operator() {
 
     openebs_cleanup_kubesystem
     logSuccess "OpenEBS CustomResourceDefinitions are ready"
+
+    # install troubleshoot spec
+    render_yaml_file_2 "$src/tmpl-troubleshoot.yaml" | kubectl apply -f -
```

Comment:
not quite the idiom we normally use, but it'll work
(I would normally render it to a file in $dst, and then include it in the kustomization)